timelock-impl/src/main/java/com/palantir/atlasdb/timelock/transaction/timestamp/ClientAwareManagedTimestampService.java

```diff
@@ -38,4 +39,6 @@
      * @return a suitable timestamp
      */
     TimestampAndPartition getFreshTimestampForClient(UUID clientIdentifier);
```

Comment:
As noted in the leased locks thing; we should be killing this method at this time (since it can be written in terms of the new one :) )